README.md

```diff
@@ -9,7 +9,7 @@
 
 ### Deployment
 
-Wesite:[Heroku](https://secret-plains-32456.herokuapp.com/)
+Wesite: [Heroku](https://secret-plains-32456.herokuapp.com/)
 
 
 ## Getting Started
```

Comment:
The getting started section of your readme file is not explicit enough. You have to describe how someone can get your project on his/her local machine. This is for the sake of people who are new to github and may not know what to do so as to have a look on your project.